dockers/docker-fpm-frr/frr/bgpd/bgpd.main.conf.j2

```diff
@@ -14,14 +14,23 @@ ip prefix-list PL_LoopbackV4 permit {{ get_ipv4_loopback_address(LOOPBACK_INTERF
 ipv6 prefix-list PL_LoopbackV6 permit {{ get_ipv6_loopback_address(LOOPBACK_INTERFACE, "Loopback0") | replace('/128', '/64') | ip_network }}/64
 {% endif %}
 !
+{# Get all interfaces that belong to vnet #}
+{% set vnet_intfs = [] %}
+{% if VLAN_INTERFACE is defined %}
+{% for (key, metadata) in VLAN_INTERFACE.iteritems() %}
+{%   if metadata.has_key("vnet_name") %}
+{%     set vnet_intfs = vnet_intfs.append(key) %}
+{%   endif %}
+{% endfor %}
+{% endif%}
 {% for (name, prefix) in VLAN_INTERFACE|pfx_filter %}
-{% if prefix | ipv4 %}
+{% if prefix | ipv4 and name not in vnet_intfs %}
```

Comment:
can we change `and name not in vnet_intfs` to `and not VLAN_INTERFACE[prefix].has_key("vnet_name")` We could remove the for loop above

Reply:
As discussed, moved to common/functions